tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -115,35 +114,33 @@ def rhsso_groups_teardown(default_sso_host):
 
 
 @pytest.fixture()
-def multigroup_setting_cleanup():
+def multigroup_setting_cleanup(default_ipa_host):
     """Adding and removing the user to/from ipa group"""
     sat_users = settings.ipa.groups
     idm_users = settings.ipa.group_users
-    ssh.command(cmd=f'echo {settings.ipa.password} | kinit admin', hostname=settings.ipa.hostname)
-    cmd = f'ipa group-add-member {sat_users[0]} --users={idm_users[1]}'
-    ssh.command(cmd, hostname=settings.ipa.hostname)
+    default_ipa_host.execute(
+        f'echo {settings.ipa.password} | kinit admin',
+    )
+    default_ipa_host.execute(f'ipa group-add-member {sat_users[0]} --users={idm_users[1]}')
     yield
-    cmd = f'ipa group-remove-member {sat_users[0]} --users={idm_users[1]}'
-    ssh.command(cmd, hostname=settings.ipa.hostname)
+    default_ipa_host.execute(f'ipa group-remove-member {sat_users[0]} --users={idm_users[1]}')
 
 
 @pytest.fixture()
-def ipa_add_user():
+def ipa_add_user(default_ipa_host):
     """Create an IPA user and delete it"""
-    result = ssh.command(
-        cmd=f'echo {settings.ipa.password} | kinit admin', hostname=settings.ipa.hostname
-    )
+    result = default_ipa_host.execute(f'echo {settings.ipa.password} | kinit admin')
     assert result.status == 0
     test_user = gen_string('alpha')
     add_user_cmd = (
         f'echo {settings.ipa.password} | ipa user-add {test_user} --first'
         f'={test_user} --last={test_user} --password'
     )
-    result = ssh.command(cmd=add_user_cmd, hostname=settings.ipa.hostname)
+    result = default_ipa_host.execute(add_user_cmd)
```

Comment:
there are many functions that add users, update, and delete users that can be added as part of the IPA host or as separate classes. this is non-blocking